docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -0,0 +1,192 @@
+| `onSubmitError`   | `error: Error`                                             |                        | Callback when feedback is unsuccessfully submitted.                                                          |
+| `onFormSubmitted` |                                                            | The form is unmounted. | Callback when the feedback form is submitted successfully, and the SuccessMessage is complete, or dismissed. |
+| `onAddScreenshot` | `attachFile: (filename: string, data: Uint8Array) => void` |                        | Callback when a screenshot is added.                                                                         |
+
+#### Screenshots
+
+The screenshot functionality is disabled by default. To enable it, pass an `imagePicker` integration library or set the `enableScreenshot` option to `true` and implement the `onAddScreenshot` callback.
+
+##### Integrating with an Image Picker Library 
+
+Currently the supported libraries are:
+- [expo-image-picker](https://docs.expo.dev/versions/latest/sdk/imagepicker/) (tested with version `16.0`)
+- [react-native-image-picker](https://github.com/react-native-image-picker/react-native-image-picker) (tested with version `7.2` and `8.0`)
+
+You just need to import the library and pass it to the `feedbackIntegration` method.
+
+```javascript
+import * as ImagePicker from 'expo-image-picker';
+```
+
+```javascript
+import * as ImagePicker from 'react-native-image-picker';
+```
+
+```javascript
+Sentry.init({
+  integrations: [
+    Sentry.feedbackIntegration({
+      imagePicker: ImagePicker,
+    }),
+  ],
+});
+```
+
+##### Implementing the `onAddScreenshot` Callback
+
+If the above libraries do not cover your use case you can manually integrate screenshots by implementing the `onAddScreenshot` callback. The callback receives a filename and a `Uint8Array` of the image data like in the example below.
+
+```javascript
+import * as Sentry from '@sentry/react-native';
+import { toByteArray } from 'react-native-quick-base64';
+import { launchImageLibrary } from 'react-native-image-picker';
+
+const handleChooseImage = (attachFile: (filename: string, data: Uint8Array) => void): void => {
+  launchImageLibrary({ mediaType: 'photo', includeBase64: true }, (response) => {
+    if (response.didCancel) {
+      console.log('User cancelled image picker');
+    } else if (response.errorCode) {
+      console.log('ImagePicker Error: ', response.errorMessage);
+    } else if (response.assets && response.assets.length > 0) {
+      const filename = response.assets[0].fileName;
+      const base64String = response.assets[0].base64;
+      const screenShotUint8Array = toByteArray(base64String);
+      if (filename && screenShotUint8Array) {
+        attachFile(filename, screenShotUint8Array);
+      }
+    }
+  });
+};
+
+Sentry.feedbackIntegration({
+    onAddScreenshot={handleChooseImage}
+});
+```
```

Comment:
To avoid users coping this instead of using the `imagePicker` option I would use a pseudo code to show how this can be integrated with other potential libs. 

````suggestion
```javascript
import * as Sentry from '@sentry/react-native';
import { getImage } from 'custom-image-picker';

const handleChooseImage = (attachFile: (filename: string, data: Uint8Array) => void): void => {
  const { filename, bytes, error, canceled } = getImage();
  if (canceled) {
  	console.log('User canceled image choice.');
  } else if (error) {
    console.log('Error during image loading', error);
  } else {
    attachFile(filename, bytes);
  }
};

Sentry.feedbackIntegration({
    onAddScreenshot={handleChooseImage}
});
```
````

Reply:
That's a good idea 👍 Thank you for the suggestion 🙇